src/phylum/ci/ci_none.py

```diff
@@ -28,17 +27,10 @@ def _check_prerequisites(self) -> None:
         """Ensure the necessary pre-requisites are met and bail when they aren't.
 
         These are the current pre-requisites for when no CI environments/platforms is detected:
-          * Run the script from the root of a git repository
+          * (None at this time)
         """
         super()._check_prerequisites()
```

Comment:
This just seems weird. Why keep something you don't need?

Reply:
`_check_prerequisites()` is an [`abstractmethod`](https://devdocs.io/python~3.11/library/abc#abc.abstractmethod) of the base class `CIBase`. It has to be defined for each child class. It just so happens that this is the only implementation (now) that has no unique pre-requisites.